DSharpPlus.Commands/Converters/DateTimeOffsetConverter.cs

```diff
@@ -11,6 +11,7 @@ namespace DSharpPlus.Commands.Converters;
 public class DateTimeOffsetConverter : ISlashArgumentConverter<DateTimeOffset>, ITextArgumentConverter<DateTimeOffset>
 {
     public DiscordApplicationCommandOptionType ParameterType { get; init; } = DiscordApplicationCommandOptionType.String;
+    public string ReadableName { get; init; } = "Date and Time";

```

Comment:
Timestamp?

Reply:
Too generalized for the common discord user